components/gitpod-db/src/typeorm/entity/db-team-subscription.ts

```diff
@@ -51,4 +51,9 @@ export class DBTeamSubscription implements TeamSubscription {
         default: false
     })
     deleted?: boolean;
+
+    @Column({
+        default: false
+    })
+    excludeFromMoreResources?: boolean;
```

Comment:
Can this really be `undefined`, given that the DB migration uses `tinyint(4) NOT NULL DEFAULT '0'`?

Reply:
Fixed and pushed.